includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -658,7 +658,7 @@ public function get_proxy_setup_url( $access_code = '', $error_code = '' ) {
 			'scope'   => rawurlencode( $scope ),
 		);
 		if ( 'missing_verification' === $error_code ) {
-			$query_args['verification_nonce'] = wp_create_nonce( 'googlesitekit_verification' );
+			$query_args['verification_nonce'] = wp_generate_password();
```

Comment:
This needs to remain as a WP nonce or it will fail verification
https://github.com/google/site-kit-wp/blob/35eef8d555bd8abe887244ae7689385dbd3ad48c/includes/Core/Authentication/Authentication.php#L419-L422